### PR TITLE
Simplify and update MathML authoring page

### PR DESCRIPTION
**Pull request by fred-wang**
#### Summary

This commit modifies the MathML authoring page to update and simplify
its content. In details:

- Align with MDN's convention: do not capitalize each word of headers,
  do not have to consecutive headers without text in between.

- Significantly reduce the amount of text in the intro (and other
  places).

- Mention distinction between inline/display math, which is an important
  aspect to be aware for math users.

- Do not mention Content MathML, it's not relevant for browsers.

- Remove section about MathML in XML, it's not critical for most web
  users these days. Use a simple info box instead.

- Remove section about email/instant messaging, split that into a simple
  info box, and add the corresponding tools to a new
  "Miscellaneous tools" section. Instantbird add-on seems unavailable
  so remove mention to it.

- Remove 'UA string sniffing' as that's not a recommended approach.
  Instead, explicitly mention it's not desired in a note and that
  the scripts use feature detection instead.

- Add notes to say that resources proposed on GitHub are open source,
  and encourage users to copy to their servers or adapt them. Also
  avoid duplicating the script/stylesheet content on MDN.

- The content of "Conversion from a simple syntax" and
  "Graphical interfaces" are unchanged for now, besides the header
  tweaks mentioned above.

- Fix heading level of "Original document information"

#### Motivation

MathML will be likely be enable in Chromium soon, potentially bringing many new users to MathML. It's important to provide up-to-date and digestible information.

Note: other improvements for future CL could be to provide a separate "MathML tutorial" page and split the authoring page into two articles (one for "Using MathML" and another for the tools review).

#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by fred-wang** [comment]
cc @sideshowbarker

File: files/en-us/web/mathml/authoring/index.md [hunk 1]
```diff
@@ -343,11 +302,17 @@ latexmlc --dest foo.html --splitat=section foo.tex
...
+### Optical character and handwriting recognitions
 
-[Inftyreader](https://www.inftyreader.org/) is able to perform some Optical Character Recognition, including translation of mathematical equations into MathML. Other tools can do handwriting recognition such as the [Windows Math Input Panel](<https://msdn.microsoft.com/library/windows/desktop/dd371698(v=vs.85).aspx>) or the online converter [Web Equation](https://webdemo.myscript.com/views/math.html).
+[Inftyreader](https://www.inftyreader.org/) is able to perform some optical character recognition, including translation of mathematical equations into MathML. Other tools can do handwriting recognition such as the [Windows Math Input Panel](<https://msdn.microsoft.com/library/windows/desktop/dd371698(v=vs.85).aspx>) or the online converter [Web Equation](https://webdemo.myscript.com/views/math.html).
 
-### Original Document Information
+## Original document information
 
 - Author(s): Frédéric Wang
 - Other Contributors: Florian Scholz
```
Comment (teoli2003) [inline files/en-us/web/mathml/authoring/index.md:318]: Do we want to remove this now that we are in GitHub and have a good history?
```suggestion
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
I propose we remove the attribution. You were the original author, and Florian is still around.

File: files/en-us/web/mathml/authoring/index.md (hunk 1 above)
Comment (fred-wang) [reply files/en-us/web/mathml/authoring/index.md:318]: I was actually wondering the same. I believe I took inspiration from https://www-archive.mozilla.org/projects/mathml/authoring but it seems I rewrote everything from scratch as the original was written during the XHTML times, so let's remove the "Original document information" (assuming @Elchi3 is also ok with that of course).

File: files/en-us/web/mathml/authoring/index.md [hunk 2]
```diff
@@ -343,12 +302,12 @@ latexmlc --dest foo.html --splitat=section foo.tex
...
+In [Thunderbird](https://www.thunderbird.net/en-US/) and [SeaMonkey](https://www.seamonkey-project.org/), you can use the **Insert HTML** command to paste any MathML content, or the **Insert Math** command to use a LaTeX-to-MathML input box. Alternatively, [MathBird](http://disruptive-innovations.com/zoo/MathBird/) provides a similar input box via AsciiMath instead.
 
-### Original Document Information
+### Optical character and handwriting recognitions
 
-- Author(s): Frédéric Wang
-- Other Contributors: Florian Scholz
-- Copyright Information: Portions of this content are © 2010 by individual mozilla.org contributors; content available under a Creative Commons license | [Details](https://www.mozilla.org/en-US/foundation/licensing/website-content/).
+[Inftyreader](https://www.inftyreader.org/) is able to perform some optical character recognition, including translation of mathematical equations into MathML. Other tools can do handwriting recognition such as the [Windows Math Input Panel](<https://msdn.microsoft.com/library/windows/desktop/dd371698(v=vs.85).aspx>) or the online converter [Web Equation](https://webdemo.myscript.com/views/math.html).
```
Comment (teoli2003) [inline files/en-us/web/mathml/authoring/index.md:313]: Typo:
```suggestion
[Inftyreader](https://www.inftyreader.org/) is able to perform some optical character recognition, including translation of mathematical equations into MathML. Other tools can do handwriting recognition such as the [Windows Math Input Panel](https://msdn.microsoft.com/library/windows/desktop/dd371698(v=vs.85).aspx) or the online converter [Web Equation](https://webdemo.myscript.com/views/math.html).
```

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**